src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,24 @@
+.catalog {
+  display: grid;
+  grid-template-columns: $card-width;
+  justify-content: center;
+  gap: $card-gap;
+
+  max-width: getMaxWidth();
+  margin: 0 auto;
```

Comment:
What does this do? Can you omit it?

Reply:
Yes, I did not notice that justify-content property has been already set and it actually places the cards in the middle of the page.